packages/core/src/Stores/notification-store.js

```diff
@@ -0,0 +1,897 @@
+            if (is_p2p_visible) {
+                this.addNotificationMessage(this.client_notifications.dp2p);
+            } else {
+                this.removeNotificationMessageByKey({ key: this.client_notifications.dp2p.key });
+            }
+
+            if (is_tnc_needed) {
+                this.addNotificationMessage(this.client_notifications.tnc);
+            }
+
+            has_missing_required_field = hasMissingRequiredField(account_settings, client, isAccountOfType);
+            if (has_missing_required_field) {
+                this.addNotificationMessage(
+                    this.client_notifications.required_fields(withdrawal_locked, deposit_locked)
+                );
+            }
+        }
+
+        if (!is_eu && isMultiplierContract(selected_contract_type) && current_language === 'EN' && is_logged_in) {
+            this.addNotificationMessage(this.client_notifications.deriv_go);
+        } else {
+            this.removeNotificationMessageByKey({ key: this.client_notifications.deriv_go.key });
+        }
+        this.setShouldShowPopups(true);
+    }
+
+    @action.bound
+    init() {
+        this.setClientNotifications();
+    }
+
+    @action.bound
+    markNotificationMessage({ key }) {
+        this.marked_notifications.push(key);
+    }
+
+    @action.bound
+    refreshNotifications() {
+        this.removeNotifications(true);
+        this.removeAllNotificationMessages();
+        this.handleClientNotifications();
+    }
+
+    @action.bound
+    removeAllNotificationMessages(should_close_persistent) {
+        this.notification_messages = should_close_persistent
+            ? []
+            : [...this.notification_messages.filter(notifs => notifs.is_persistent)];
+    }
+
+    @action.bound
+    removeNotifications(should_close_persistent) {
+        this.notifications = should_close_persistent
+            ? []
+            : [...this.notifications.filter(notifs => notifs.is_persistent)];
+    }
+
+    @action.bound
+    removeNotificationByKey({ key }) {
+        this.notifications = this.notifications.filter(n => n.key !== key);
+    }
+
+    @action.bound
+    removeNotificationMessage({ key, should_show_again } = {}) {
+        if (!key) return;
+        this.notification_messages = this.notification_messages.filter(n => n.key !== key);
+        // Add notification messages to LocalStore when user closes, check for redundancy
+        const active_loginid = LocalStore.get('active_loginid');
+        if (!excluded_notifications.includes(key) && active_loginid) {
+            const messages = LocalStore.getObject('notification_messages');
+            // Check if same message already exists in LocalStore for this account
+            if (messages[active_loginid] && messages[active_loginid].includes(key)) {
+                return;
+            }
+            const current_message = () => {
```

Comment:
```suggestion
            const currentMessage = () => {
```
Functions naming standard